### PR TITLE
new dallas chapter

### PR DESCRIPTION
**Pull request by Neetuj**
I have currently added my email address. I will check with sheila and update it once we have a dallas@pyladies.com email

### CODE REVIEW

**Comment by Neetuj** [comment]
@sarasafavi  please review

File: www/locations/dallas/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <section id="archive">
+        <h1>PyLadies dfw(dallas)</h1>
+            <article>
+                <ul class="social">
+                    <aside>
+
++                        <h4><a href="{{ get_url("locations/dallas") }}">Dallas</a> |
++                            <a class="social icon vcard" data-icon="&#59170;" href="mailto:nutshi@gmail.com" title"Contact"></a>
++                            <a class="social icon facebook" data-icon="&#62221;" target="_blank" href="httphttps://www.facebook.com/DFWPyLadies" title="Facebook"></a>
++                            <a class="social icon meetup" data-icon="&#59172;" target="_blank" title="Twitter" href="https://www.facebook.com/DFWPyLadieshttp://www.meetup.com/DFW-PyLadies/"></a>
```
Comment (sarasafavi) [inline www/locations/dallas/index.html]: typo in href on both L12 & L13

**Comment by sarasafavi** [comment]
@Neetuj LGTM aside from comment above... I'll email you now but FYI you'll need to wait on a maintainer (not me) to merge this in :)

**Comment by Neetuj** [comment]
I understand.. thanks!

**Comment by econchick** [comment]
@Neetuj is this the same as the Dallas-Fort Worth chapter?  Because if so, you already have dfw@pyladies.com set up.  Did you want to switch it to dallas@?

**Comment by econchick** [comment]
also @Neetuj fix your change, rebase, and push it again. The PR will automatically update.

**Comment by econchick** [comment]
Last question! @Neetuj did you want dallas.pyladies.com (or dfw) and do you want that subdomain to redirect to www.pyladies.com/locations/dallas ? at least for now until you have something set up?

**Comment by Neetuj** [comment]
Thank for letting me know. Yes this is the same chapter.

**Comment by Neetuj** [comment]
changed the email address.

**Comment by econchick** [comment]
Let me know when you fix those two issues that @sarasafavi pointed out on lines 12 & 13.

**Comment by Neetuj** [comment]
which 2 issues ?
I already changed the email.

**Comment by econchick** [comment]
I'm referring to the two mangled hrefs that Sara pointed out.

**Comment by Neetuj** [comment]
ahh .. totally missed the comment and the error .. thanks!

File: www/config.yml [hunk 2]
```diff
@@ -326,3 +326,12 @@ chapters:
       twitter: pyladiesdc
       meetup: dc-pyladies
 
+    - name: Dallas, TX
```
Comment (econchick) [inline www/config.yml]: For some reason, you have `+`s on every line.

Also, please keep this in alphabetical order.  Dallas should be placed after Chicago.

File: www/locations/dallas/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{% extends "site.html" %}
+{% block title %}PyLadies dfw(dallas){% endblock %}
+{% block content %}
+    <section id="archive">
+        <h1>PyLadies dfw(dallas)</h1>
+            <article>
+                <ul class="social">
+                    <aside>
+
++                        <h4><a href="{{ get_url("locations/dallas") }}">Dallas</a> |
```
Comment (econchick) [inline www/locations/dallas/index.html]: Between lines 10 - 14, you have extra `+`s which render like so: 
2015-04-28 at 8 26 am

Please remove those `+`s.

File: www/config.yml [hunk 4]
```diff
@@ -129,6 +129,14 @@ chapters:
       email: chicago@pyladies.com
       twitter: PyLadiesChicago
       meetup: Chicago-PyLadies
+  
+    - name: Dallas, TX
+      meetup_id: 2292131
```
Comment (sheilatron) [inline www/config.yml]: Where does this ID come from?

Comment (econchick) [inline www/config.yml]: Sheila - it's difficult to find. You have to go to the dev site of Meetup and use the console to do an API request using the "groups" endpoint. Let me know if you want further instruction.

Side note: a script will be added to the pyladies package to easily make that API request:)

Fat fingered from my iPhone

**Comment by econchick** [comment]
Sorry it took me a while - this is merged and deployed. Thanks!

**Pull request merged**